exp-player/addon/components/exp-frame-base.js

```diff
@@ -85,10 +87,11 @@ export default Ember.Component.extend({
             this.set('eventTimings', timings);
         },
         next() {
-            console.log('Leaving frame ID', this.get('id'));
+            var frameId = `${this.get('frameIndex')}-${this.get('id')}`;
```

Comment:
To some extent, tampering with the IDs is necessary, particularly if the experiment uses the same frame (ID) at two different points in the experiment.  (we need a way to disambiguate between two data points with the same basic name)

As for the order? It was a readability thing. Objects keys aren't guaranteed to appear in a particular order, but by putting the number first (and pretty-printing), you are more likely to get things in time order, which helps to read the flow of the experiment...

```
1-consent-form
2-sound-check
3-consent-form
```
rather than
```
consent-form-1
consent-form-3
sound-check-2
```

(the 11th frame will still suck, but c'est la vie)